### PR TITLE
Adding builder-ubi-buildpackless-base on integration tests

### PR DESCRIPTION
**Pull request by pacostas**
## Summary

This PR adds the `builder-ubi-buildpackless-base` builder to run against all the integration tests of yarn.

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [x] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: integration/layer_reuse_test.go [hunk 1]
```diff
@@ -25,6 +25,9 @@ func testRebuildLayerReuse(t *testing.T, context spec.G, it spec.S) {
 
 		name   string
 		source string
+
+		pullPolicy = "never"
+		// extenderBuildStr = ""
```
Comment (mhdawson) [inline integration/layer_reuse_test.go:30]: Was this comment out intentional?  Same below

Comment (pacostas) [reply integration/layer_reuse_test.go]: removed!

File: integration/default_test.go [hunk 2]
```diff
@@ -68,16 +74,21 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
...
 			image, logs, err = pack.WithNoColor().Build.
-				WithPullPolicy("never").
-				WithBuildpacks(buildpack, buildPlanBuildpack).
+				WithExtensions(
+					settings.Extensions.UbiNodejsExtension.Online,
+				).
+				WithBuildpacks(
+					settings.Buildpacks.Yarn.Online,
+					settings.Buildpacks.BuildPlan.Online,
+				).
```
Comment (mhdawson) [inline integration/default_test.go:83]: Asked @pacostas about this change, it was done to be consistent with all of the other buildpack integration tests.

File: integration/default_test.go [hunk 3]
```diff
@@ -68,16 +74,21 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
...
 				WithSBOMOutputDir(sbomDir).
 				WithEnv(map[string]string{"BP_LOG_LEVEL": "DEBUG"}).
+				WithPullPolicy(pullPolicy).
 				Execute(name, source)
 			Expect(err).ToNot(HaveOccurred(), logs.String)
 
 			// Ensure yarn is installed correctly
-
-			container, err = docker.Container.Run.WithCommand("which yarn").Execute(image.ID)
+			container, err = docker.Container.Run.WithCommand("command -v yarn").Execute(image.ID)
```
Comment (mhdawson) [inline integration/default_test.go:91]: I asked @pacostas about this, we don't install which by default in the ubi stack, this allows the test to run for both ubi and jammy

File: integration/default_test.go [hunk 4]
```diff
@@ -155,8 +166,13 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 			)
 
 			image, logs, err = pack.WithNoColor().Build.
-				WithPullPolicy("never").
-				WithBuildpacks(buildpack, buildPlanBuildpack).
+				WithExtensions(
```
Comment (mhdawson) [inline integration/default_test.go:169]: Similar change to previous for consistenccy with other buildpack integration tests.

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by pacostas** [comment]
@mhdawson I added this change https://github.com/paketo-buildpacks/yarn/compare/0dc16e8f1274486fc2de415091428f0a2b097f48..fcd93be671c8ca8a1ce4adef9e24201562a5ac51 Practically is skipping the offline tests, as we cant support them at the moment. Can you LGTM that too?

**Review by mhdawson — APPROVED** [review]
LGTM again

**Review by thitch97 — APPROVED** [review]

**Pull request merged**